# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/ActorsParent.cpp
# Commit: 5b13eeec7485
# Full Hash: 5b13eeec7485199e1bfab8744cd683706ed1db3f
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2020-01-06 21:54:03
# Regressor Bug: 1600906
# File Overlap Count: 1
# Description:
#   Bug 1600906 - Merge identical switch cases. r=dom-workers-and-storage-reviewers,ttung
#   
#   Depends on D57986
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D57987
# ==============================================================================

diff -r 498d5e653e97 -r 5b13eeec7485 dom/indexedDB/ActorsParent.cpp
--- a/dom/indexedDB/ActorsParent.cpp	Fri Dec 20 15:32:36 2019 +0000
+++ b/dom/indexedDB/ActorsParent.cpp	Fri Dec 20 15:33:40 2019 +0000
@@ -24324,8 +24324,6 @@
       break;
 
     case PreprocessResponse::TObjectStoreGetPreprocessResponse:
-      break;
-
     case PreprocessResponse::TObjectStoreGetAllPreprocessResponse:
       break;
 
